### DISCUSSION TITLE
Pimcore Version Upgrade from 6.3.x to 6.9.x

### DISCUSSION THREAD

**Discussion by ARY112** [pimcore/pimcore#11947] [Q&A / Support] [answered]
Hello ,

We are using Pimcore PIM module and using version 6.3.6.

We want to upgrade to version 10 but before that as per documentation we need to upgrade to 6.9 then version 10.

- If you're on a version between 6.0.0 and 6.8.x, then please update to 6.9.x first and then continue with the upgrade steps.

Can we directly migrate to version 6.9.x by making change in composer.json or we need to migrate one by one between the versions 6.3.6 and 6.9.x.

Kindly advice.

**Comment by dvesh3** [comment, marked as answer]
@ARY112 I would suggest to go through the list of [upgrade notes](https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html) until 6.9 and consider adapting the required changes, if it is feasible then you can directly bump to 6.9 version without a problem.
